### ISSUE TITLE
Crackpack won't open worlds anymore

### ISSUE THREAD

**Issue by TheNeutral** [sciguyryan/TheCrackPack#985]
I was playing on the Crackpack for a while with little to no issues, but recently I updated Botania and afterwards it began to stop loading my worlds. I thought the update was the source of the issue so I got rid of the new version and reinstalled the pack. The issue still remains. The main screen will load fine but when I select my world it will load for a few seconds with no text on screen and then shut off and send me back to the launcher home.

**Comment by TheNeutral** [comment]
Here is the log:
http://paste.atlauncher.com/view/f1441782

**Comment by evknucklehead** [comment]
Delete usercache.json from the instance folder.

**Comment by TheNeutral** [comment]
I deleted that file and it doesn't close MC now. Instead, it will begin loading the world and send me back to the MC title screen after displaying the loading terrain text for a few moments.
Here is the log: 
http://paste.atlauncher.com/view/a2c311f4

**Comment by evknucklehead** [comment]
When you reinstalled the pack, you forgot to include a couple of the optional mods, specifically Ars Magica and Tails. Also, since you had updated Botania and Fastcraft, you'll need to update those again to avoid the error message.

**Comment by TheNeutral** [comment]
Ok, so all that's been done and it ran very smoothly for a while but now I am having another response issue. It loads and places me in the game with my HUD displayed and a bit of terrain loaded but after a bit, it won't respond. I tried waiting, but after it catches up it goes back into not responding permanently . 
Log: http://paste.atlauncher.com/view/fbe781c2

**Comment by evknucklehead** [comment]
Turn down your memory allocations. There's no reason to raise the minimum at all, and you're not leaving enough for the system if you turn the maximum up to such a high percentage of the total. Try putting minimum at 256M and maximum at 4GB and see if it still has problems.

**Comment by TheNeutral** [comment]
It let me play again for a total of 25 min. Then it went back to not responding. I notice in the game that my framerate is good, but when I go to new areas it has a difficult time catching me up in those new areas. It takes a while for the chunks to come in so, I am sat in a void for a while before they actually load. This is normally when the game quits responding.

Log: http://paste.atlauncher.com/view/1839009f

**Comment by evknucklehead** [comment]
Yeah, version 1.7.10 is notoriously slow at world generation, and adding all the mods the pack does only adds to the problem. On the plus side, the version of Forge included in pack 2.0.3.0 does have some improvements in that regard.

If you're teleporting a large distance, like it says in your report, it has to generate the whole visible range before you can do anything. If you cut your view distance to 8 chunks or less, it has much less it needs to generate at a time, so that will help a lot with your lag spikes. 

Also, when I said set the maximum at 4, I meant 4, not 3.5. Giving the pack 4 GB is about right for most situations. As long as you're leaving at least 1.5-2GB for the system to do its work, you should be fine with a higher maximum allocation.
